mlos_bench/mlos_bench/storage/sql/experiment.py

```diff
@@ -22,7 +22,7 @@
 from mlos_bench.storage.sql.schema import DbSchema
 from mlos_bench.storage.sql.trial import Trial
 from mlos_bench.tunables.tunable_groups import TunableGroups
-from mlos_bench.util import utcify_timestamp
+from mlos_bench.util import try_parse_val, utcify_timestamp
```

Comment:
Work to return numeric values for metrics and configs if possible.

TODO: Split this change out to a separate PR.